pkg/types/types.go

```diff
@@ -98,6 +98,10 @@ type Advisory struct {
 	VulnerabilityID string   `json:",omitempty"` // CVE-ID or vendor ID
 	VendorIDs       []string `json:",omitempty"` // e.g. RHSA-ID and DSA-ID
 
+	// Rpm packages have advisories for different architectures with same package name
+	// This field is required to separate these packages.
+	Arch string `json:"-"`
```

Comment:
according to tests it may have values like `aarch64|ppc64le|x86_64`. I think it's better to store it as a collection to avoid any future misuse